e2e/update/src/generic-tests.ts

```diff
@@ -0,0 +1,50 @@
+import { browser } from 'protractor';
+import { BaseDevUpdateTestPage } from './update.po';
+
+const page = new BaseDevUpdateTestPage();
+
+export function onPageLoadTest(PAGE_URL: string) {
```

Comment:
Minor point, but I would remove the "on" from each of these tests.  So it'd be "pageLoadTest()"

You might also want to prefix (rather than postfix/append) "test", so it'd be "testPageLoad(), testClickPrevStepper()" Not only does it make it clear the functions are part of the same "family", but it also makes the language imperative like you're giving an order to the tests (test this! test that!)